libraries/golang/layering/models/silos.go

```diff
@@ -4,3 +4,27 @@ type Silo struct {
 	Id string `db:"id"`
 	State string `db:"state"` // TODO: Decide possible state values and replace with an enum. (active, disabled, deleted?)
 }
+
+type Node struct {
```

Comment:
Put these in a new file in grpc/internal/transformations. Use FxnArgs struct as args